### PR TITLE
feat: add cromwell support

### PR DESCRIPTION
**Pull request by dapineyro**
## Jira

https://lifebit.atlassian.net/browse/DEL-8956
https://lifebit.atlassian.net/browse/DEL-9037

## Overview

This PR is the first step to add WDL pipeline support. WDL pipelines run using a Cromwell server that has to be manually managed in CloudOS. The present PR allows to check the Cromwell server status, restart and stop it.

> NOTE: this PR has to be reviewed after https://github.com/lifebit-ai/cloudos-py/pull/49 merging

## Changes

- Adds `cromwell` command with the following subcommands:
    * status: to check current Cromwell status
    * start: to restart Cromwell server in case it is stopped
    * stop: to stop Cromwell server in case it is running
 - New `REQUEST_INTERVAL` global var to parametrise interval time between status requests. This is used by `cloudos job run --wait-completion` and `cloudos cromwell restart` commands.
 - `clos.Cloudos` object now accepts three arguments: `cloudos_url`, `apikey` and `cromwell_token`. This is required as `apikey` is different from `cromwell_token`. That required modification of each of the calls in `__main__.py`
 - Version bumped to 0.1.1
- CHANGELOG.md, README.md and _version.py updated accordingly

## Test

In a GEL CloudOS Jupyter session:

```bash
# Cromwell server requires its particular token
CLOUDOS="xxxx"
CROMWELL_TOKEN="xxxx"
WORKSPACE_ID="xxxx"

docker run --rm -it quay.io/lifebitaiorg/cloudos-py:v0.1.1

# Check Cromwell status
cloudos cromwell status \
    -c $CLOUDOS \
    -t $CROMWELL_TOKEN \
    --workspace-id $WORKSPACE_ID
```

```console
CloudOS python package: a package for interacting with CloudOS.

Version: 0.1.1

Cromwell server functionality: check status, start and stop.

Executing status...
	Current Cromwell server status is: Stopped
```

```bash
# Cromwell restart
cloudos cromwell start \
    -c $CLOUDOS \
    -t $CROMWELL_TOKEN \
    --workspace-id $WORKSPACE_ID
```

```console
CloudOS python package: a package for interacting with CloudOS.

Version: 0.1.1

Cromwell server functionality: check status, start and stop.

Starting Cromwell server...
	Current Cromwell server status is: Initializing

	Current Cromwell server status is: Running
```

```bash
# Cromwell stop
cloudos cromwell stop \
    -c $CLOUDOS \
    -t $CROMWELL_TOKEN \
    --workspace-id $WORKSPACE_ID
```

```console
CloudOS python package: a package for interacting with CloudOS.

Version: 0.1.1

Cromwell server functionality: check status, start and stop.

Stopping Cromwell server...
	Current Cromwell server status is: Stopped
```

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,13 @@
 ## lifebit-ai/cloudos-py: changelog
 
+### 0.1.1 - 2022-07-12
+- Adds WDL pipeline support, iteration 1: cromwell server managing.
+Now, a new command `cloudos cromwell` is available, with the following
+subcommands:
+    * status
+    * restart
```
Comment (cgpu) [inline CHANGELOG.md:8]: @dapineyro let's mirror the UI and name it `start` 
image

File: cloudos/__main__.py [hunk 2]
```diff
@@ -12,6 +12,7 @@
 JOB_COMPLETED = 'completed'
 JOB_FAILED = 'failed'
 JOB_ABORTED = 'aborted'
+REQUEST_INTERVAL = 60
```
Comment (cgpu) [inline cloudos/__main__.py:15]:
```Suggestion
REQUEST_INTERVAL = 30
```

to make it slightly faster for the user to get the restart.

**Review by cgpu — CHANGES_REQUESTED** [review]

**Comment by dapineyro** [comment]
@cgpu . Thanks for all your suggestions. All the changes were made. Please, take a look when you got time.

File: README.md [hunk 3]
```diff
@@ -257,7 +258,7 @@ Executing status...
 You can get a summary of your last 30 submitted jobs in two different formats:
 
 - CSV: this is a table with a minimum predefined set of columns by default, or all the
-available columns using the `--full-data` parameter.
+available columns using the `--all-fields` argument.
```
Comment (cgpu) [inline README.md:261]: nice!

**Pull request merged**